tests/vm_console_proxy_test.go

```diff
@@ -0,0 +1,177 @@
+package tests
+
+import (
+	"reflect"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	apps "k8s.io/api/apps/v1"
+	core "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/pointer"
+)
+
+var _ = Describe("VM Console Proxy Operand", func() {
+	var (
+		clusterRoleResource        testResource
+		clusterRoleBindingResource testResource
+		serviceAccountResource     testResource
+		serviceResource            testResource
+		deploymentResource         testResource
+		configMapResource          testResource
+	)
+
+	BeforeEach(func() {
```

Comment:
If I understand correctly, [here](https://github.com/kubevirt/ssp-operator/blob/9973ad5d20d778652925281f3d4072ba75e00c64/tests/tests_suite_test.go#L115) we already have relevant annotations, and in case they are not there, we should add them here in `vm_console_proxy_test.go`?

Reply:
Correct. The linked part of code only adds the annotations in the `newSspStrategy`. If the tests are run with an existing SSP operator deployment (as is done by QE), then the annotations may not be there.